unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/WorldChatWindowHUDView.cs

```diff
@@ -67,7 +70,18 @@ public void ActivatePreview()
         OnActivatePreview?.Invoke();
     }
 
-    public void OnPointerClick(PointerEventData eventData) { DeactivatePreview(); }
+    public void OnPointerClick(PointerEventData eventData) {
+        if (eventData.button == PointerEventData.InputButton.Left)
+        {
+            var results = new List<RaycastResult>();
+            EventSystem.current.RaycastAll(eventData, results);
+            if (results.Any(r => r.gameObject.Equals(chatHudView.gotoPanel.container.gameObject))) 
```

Comment:
whats the purpose of this addition?

Reply:
None, was a leftover from the previous implementation of this feature, removed